src/node_contextify.cc

```diff
@@ -132,41 +133,51 @@ class ContextifyContext {
         return;
 
       if (!has.FromJust()) {
-        // Could also do this like so:
-        //
-        // PropertyAttribute att = global->GetPropertyAttributes(key_v);
-        // Local<Value> val = global->Get(key_v);
-        // sandbox->ForceSet(key_v, val, att);
-        //
-        // However, this doesn't handle ES6-style properties configured with
-        // Object.defineProperty, and that's exactly what we're up against at
-        // this point.  ForceSet(key,val,att) only supports value properties
-        // with the ES3-style attribute flags (DontDelete/DontEnum/ReadOnly),
-        // which doesn't faithfully capture the full range of configurations
-        // that can be done using Object.defineProperty.
-        if (clone_property_method.IsEmpty()) {
-          Local<String> code = FIXED_ONE_BYTE_STRING(env()->isolate(),
-              "(function cloneProperty(source, key, target) {\n"
-              "  if (key === 'Proxy') return;\n"
-              "  try {\n"
-              "    var desc = Object.getOwnPropertyDescriptor(source, key);\n"
-              "    if (desc.value === source) desc.value = target;\n"
-              "    Object.defineProperty(target, key, desc);\n"
-              "  } catch (e) {\n"
-              "   // Catch sealed properties errors\n"
-              "  }\n"
-              "})");
-
-          Local<Script> script =
-              Script::Compile(context, code).ToLocalChecked();
-          clone_property_method = Local<Function>::Cast(script->Run());
-          CHECK(clone_property_method->IsFunction());
+        Local<Object> desc_from_vm_context =
+            global->GetOwnPropertyDescriptor(context, key)
+            .ToLocalChecked().As<Object>();
+
+        bool isAccessor = desc_from_vm_context
+            ->Has(context, env()->get_string())
+            .FromJust() || desc_from_vm_context
+                ->Has(context, env()->set_string())
+                .FromJust();
```

Comment:
This is mostly just personal preference but it's kind of a run-on line of code that's arguably more legible when written like this:
```c++
bool is_accessor =
    desc_from_vm_context->Has(context, env()->get_string())
    .FromJust() ||
    desc_from_vm_context->Has(context, env()->set_string())
    .FromJust();
```
If you shorten `desc_from_vm_context` you can probably even make it fit on fewer lines.

But whatever you do, the convention in core is to break after the '=' when the RHS doesn't fit on a single line.  Can you update that here and below?